plugins/reach/lib/model/EntryVendorTask.php

```diff
@@ -460,11 +460,14 @@ public function addSchedulingData()
 				$taskData = $this->getTaskJobData();
 				/* @var $connectedEvent LiveStreamScheduleEvent */
 				$connectedEvent = $taskData->getScheduleEvent();
+				/* @var $catalogItem VendorLiveCaptionCatalogItem */
+				$catalogItem = $this->getCatalogItem();
 
 				$feature = new LiveCaptionFeature();
 				$feature->setPreStartTime($connectedEvent->getStartDate(null) - $taskData->getStartDate());
 				$feature->setPostEndTime($taskData->getEndDate() - $connectedEvent->getEndDate(null));
 				$feature->setSystemName(LiveCaptionFeature::defaultName(LiveFeature::REACH_FEATURE_PREFIX . "-{$this->getId()}"));
+				$feature->setLanguage($catalogItem->getSourceLanguage());
```

Comment:
Better convert here to 3 letter code by:
languageCodeManager::getLanguageCode($catalogItem->getSourceLanguage(), true);
And then add in the comments for all liveCaptionFeature language that it is 3 letter code